dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java

```diff
@@ -204,10 +205,9 @@ private Http2FrameCodec buildHttp2FrameCodec(URL url) {
     }
 
     private TripleConfig getTripleConfig(URL url) {
-        return url.getOrDefaultApplicationModel()
-                .getApplicationConfigManager()
-                .getProtocol(url.getProtocol())
-                .orElseThrow(() -> new IllegalStateException("There's no ProtocolConfig specified."))
-                .getTriple();
+        return url.getOrDefaultApplicationModel().getApplicationConfigManager().getDefaultProtocols().stream()
```

Comment:
getApplicationConfigManager().getProtocol(url.getProtocol())?
Using findFirst is not reasonable. What if the first one is the Dubbo protocol? Also, if nothing is found, it should throw an exception; otherwise, how would the initial values be initialized?

Reply:
If this application is a pure consumer, no ProtocolConfig will be configured, but if it needs to call the downstream triple service, configClientPipeline requires a TripleConfig.

Reply:
Updated:
```java
    private TripleConfig getTripleConfig(URL url) {
        return url.getOrDefaultApplicationModel()
                .getApplicationConfigManager()
                .getOrAddProtocol(url.getProtocol())
                .getTriple();
    }
```